system-tests/projects/no-specs-custom-pattern/.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Three new system-test projects were added in this PR. Lots of files, some duplication, but utilizing these gives us as much e2e coverage as possible (in comparison to config interceptions).